GhostNetwork.Publications.Api/Controllers/CommentController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using GhostNetwork.Publications.Api.Models;
+using GhostNetwork.Publications.Domain;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GhostNetwork.Publications.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class CommentController : ControllerBase
+    {
+        private readonly ICommentService commentService;
+
+        public CommentController(ICommentService commentService)
+        {
+            this.commentService = commentService;
+        }
+
+        [HttpPost]
+        [ProducesResponseType(StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<ActionResult<Comment>> CreateAsync([FromBody] CreateCommentModel model)
+        {
+            var id = await commentService.CreateAsync(model.PublicatioId, model.Content, model.ReplyCommentId);
```

Comment:
Use deconstructing tuples to avoid Item1 and Item2 naming
Fix typo **PublicatioId**